addon/ng2/blueprints/ng2/files/angular-cli.json

```diff
@@ -5,9 +5,24 @@
   },
   "apps": [
```

Comment:
I'm not sure what the plans are for supporting library generation with the CLI, but I was thinking that `apps` might be better named `modules` with  a `type` field for each module definition which would correlate to a set of webpack configs for that type (e.g., `app` or `lib`).